CRM/Report/Form/Contact/Relationship.php

```diff
@@ -507,9 +504,13 @@ public function statistics(&$rows) {
         //for displaying relationship type filter
         if ($value['title'] == 'Relationship') {
           $relTypes = CRM_Core_PseudoConstant::relationshipType();
-          $statistics['filters'][$id]['value'] = 'Is equal to ' .
-            $relTypes[$this->_params['relationship_type_id_value']]['label_' .
-            $this->relationType];
+          $op = 'Is one of ';
```

Comment:
@seamuslee001 This string needs to be conditional on the filter operator (is one of, is not one of). I think t also needs to be translatable. So replace w/ something like this:
```
          if (CRM_Utils_Array::value('relationship_type_id_op', $this->_params) == 'in') {
            $op = ts('Is one of') . ' ';
          }
          else {
            $op = ts('Is not one of') . ' ';
          }
```